extensions/amp-analytics/0.1/vendors.js

```diff
@@ -63,6 +63,21 @@ export const ANALYTICS_CONFIG = {
           '${baseSuffix}'
     },
     'optout': '_gaUserPrefs.ioo'
+  },
+
+  'snowplow': {
+    'transport': {'beacon': false, 'xhrpost': false, 'image': true},
+    'requests': {
+      'aaVersion': 'amp-0.1',
```

Comment:
Your call but would make sense to me to have the tracker name/version available to all vendors...